Tests/FluentAssertions.Specs/OccurrenceConstraintSpecs.cs

```diff
@@ -25,17 +28,24 @@ public class OccurrenceConstraintSpecs
         { AtMost.Thrice(), 3 },
         { AtMost.Times(4), 3 },
         { AtMost.Times(4), 4 },
+        { 4.TimesOrLess(), 4 },
+        { 4.TimesOrLess(), 1 },
         { Exactly.Once(), 1 },
         { Exactly.Twice(), 2 },
         { Exactly.Thrice(), 3 },
         { Exactly.Times(4), 4 },
+        { 4.TimesExactly(), 4 },
         { LessThan.Twice(), 1 },
         { LessThan.Thrice(), 2 },
         { LessThan.Times(4), 3 },
+        { 4.TimesOrLess(), 4 },
+        { 4.TimesOrLess(), 1 },
         { MoreThan.Once(), 2 },
         { MoreThan.Twice(), 3 },
         { MoreThan.Thrice(), 4 },
         { MoreThan.Times(4), 5 },
+        { 4.TimesOrMore(), 4 },
+        { 4.TimesOrMore(), 5 },
```

Comment:
These cases are already included above.
Same for `FailingConstraints`.

Reply:
Thanks for the hint.